### PR TITLE
Add security policy to repository

### PR DESCRIPTION
**Pull request by izgeri**
Adds a SECURITY.md that more clearly states the policy for reporting
security issues in this repository.

### What ticket does this PR close?
n/a

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Follow-on issues
- [ ] Follow-up issue(s) have been logged (and links included below) to update documentation or related projects, or
- [x] No follow-up issues are required

### CODE REVIEW

**Comment by izgeri** [comment]
@andytinkham I took your suggestion and revised the response window. Can you please re-review? Thanks!

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+When the security team receives a security bug report, they will assign it to a
+primary handler. This person will coordinate the fix and release process,
+involving the following steps:
+
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    released as fast as possible.
+
```
Comment (andytinkham) [inline SECURITY.md]: It seems like there's a few more steps here - we're not actually disclosing anything in this section. Do we need to include things like getting a CVE number, registering the vulnerability with GitHub, communicating to our customers the need to upgrade, and so on? Maybe this is contextual enough that the step is just something like `* Communicate the vulnerability publicly through the appropriate channels as determined by the vulnerability`.

```suggestion
  * Communicate the vulnerability and the need to upgrade to a fixed version publicly in appropriate channels
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+The CyberArk Conjur team and community take all security bugs in Conjur seriously.
+Thank you for improving the security of Conjur. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the lead maintainers at security@conjur.org.
+
+The maintainers will acknowledge your email within 2 business days, and will send a
+more detailed response within 2 business days indicating the next steps in handling
```
Comment (andytinkham) [inline SECURITY.md]:
```suggestion
The maintainers will acknowledge your email within 2 business days. Subsequently, we will 
send a more detailed response within 2 business days of our acknowledgement indicating the next steps in handling
```

File: SECURITY.md (hunk 1 above)
Comment (izgeri) [reply SECURITY.md]: I know that we are working on defining more clearly what the next steps are for communication in a situation like this, and I definitely agree we should update this file to be absolutely clear to community members what that policy is once we've defined it. But for now, I think adding this comment is premature and we should hold off.

**Pull request merged**